internal/envoy/v3/socket_test.go

```diff
@@ -27,11 +27,11 @@ import (
 
 func TestUpstreamTLSTransportSocket(t *testing.T) {
 	tests := map[string]struct {
-		ctxt *envoy_transport_socket_tls_v3.UpstreamTlsContext
+		ctx  *envoy_transport_socket_tls_v3.UpstreamTlsContext
```

Comment:
there are many possible ways to abbreviate context, but this short form dominates this repository (and happens to be my preference)